test/artifact_bucket_test.go

```diff
@@ -48,7 +48,7 @@ func TestStorageBucketPipelineRun(t *testing.T) {
 		t.Skip("GCP_SERVICE_ACCOUNT_KEY_PATH variable is not set.")
 	}
 	c, namespace := setup(t)
-	t.Parallel()
+	// Bucket tests can't run in parallel without causing issues with other tests.
```

Comment:
what kind of issues?

Reply:
The `TestPipelineRun` tests hang for me when run in parallel with `TestStorageBucketPipelineRun` - my guess is that when they get created, the `ConfigMap` is set up to use a bucket, but by the time they actually get going, the bucket has been deleted due to this test finishing.